definitions/features/pulpcore.rb

```diff
@@ -15,6 +16,10 @@ class Features::Pulpcore < ForemanMaintain::Feature
     end
   end
 
+  def cli_available?
```

Comment:
I guess, design wise, this should be moved to an own feature, but I didn't want to refactor all of this in a bugfix PR.